Stylesheets/common_css/general.css

```diff
@@ -0,0 +1,28 @@
+
+.container{
+    margin: 2rem auto;
+    min-width: 15rem;
+    max-width: 25rem;
+    min-height: 5rem;
+    max-height: 10rem;
+    background-color: var( --st-background-grey);
+    border: 2px solid grey;
+}
+.padding-between{
+    padding:0.5rem 0.5rem;
```

Comment:
`padding: 0.5rem 0.5rem` is same as `padding: 0.5rem;`, so you can change that

Reply:
Hi Krituraj,
I have changed it.
Thanks for pointing it out